com.abagames.noiz2sa.json

```diff
@@ -0,0 +1,135 @@
+{
+  "app-id": "com.abagames.noiz2sa",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "noiz2sa",
+  "rename-desktop-file": "noiz2sa.desktop",
+  "rename-icon": "noiz2sa",
+  "rename-appdata-file": "noiz2sa.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--persist=.noiz2sa.prf"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/SDL/SDL-1.2.15.json",
+    "shared-modules/SDL/SDL_mixer-1.2.12.json",
+    {
+      "name": "bulletml",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://http.debian.net/debian/pool/main/b/bulletml/bulletml_0.0.6.orig.tar.gz",
+          "sha256": "162544e708cd8f61c944f7eac2a0403d2900d29fa9b1dc45235cfccfac8ae960"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/fixes.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/bulletml_d.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/d_cpp.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/warnings.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/makefile.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/includes.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/bulletml/get-rid-of-boost.patch"
+        }
+      ],
+      "build-commands": [
+        "rm -f src/calc.cpp",
+        "rm -f src/*.o src/*/*.o",
+        "make -C src CXXFLAGS=\"$(CXXFLAGS) $(CPPFLAGS)\" libbulletml.a",
+        "rm -f src/*.o src/*/*.o",
```

Comment:
`make clean` maybe?

Reply:
Again, I copied these instructions from Debian, but apparently `make clean` does the same...